python/lsst/sims/maf/utils/bundleList2Dicts.py

```diff
@@ -0,0 +1,51 @@
+
+__all__ = ['bundleMatch', 'bundleList2Dicts']
+
+def bundleMatch(bundle1, bundle2):
+    """
+    Check if 2 metric bundle objects are compatible to be in the same
+    metricBundleGroup.
+    """
+    result = True
+    if bundle1.sqlconstraint != bundle2.sqlconstraint:
+        return False
+    for stacker1 in bundle1.stackerList:
+        for stacker2 in bundle2.stackerList:
+            if stacker1.__class__.__name__ == stacker2.__class__.__name__:
+                if stacker1 != stacker2:
+                    return False
+
+    # Possible bug in slicer comparison, with != and == giving same results
+    if bundle1.slicer == bundle2.slicer:
+        pass
+    else:
+        return False
+
+    return result
+
+
+def list2Dict(inlist):
+    result = {}
+    for i,oneList in enumerate(inlist):
+        result[i] = oneList
+    return result
+
+def bundleList2Dicts(bundleList):
+    """
```

Comment:
Did you see the method in the metricBundleGroup file called "makeBundleDict" (makes a dict with keys set by the 'fileroot', from a list)? Pretty much the same thing as list2Dict, but the keys are not simple numbers. 

I see that we have several utilities related to metricBundles floating around. Let's consolidate 
- emptyMetricBundle  (rename it as "createEmptyMetricBundle" so it's clear it's a utility?)
- makeBundleDict (rename it to "makeBundleDictFromList" ? )
- bundleList2Dicts (rename it to "makeBundleGroupsFromList" -- and maybe have it return actual metricBundleGroups instead of the dictionary of dictionaries? (it would be a dictionary of metricBundleGroups)
- bundleMatch (rename to "checkBundleMatches" or something? it's not doing a match .. it's checking if they do)

and then put all of these into a module here (remove from the metricBundle* files) and call this file/module metricBundleUtils ?